state/factory/workingset.go

```diff
@@ -151,6 +157,9 @@ func (ws *workingSet) runAction(
 	if protocol.MustGetBlockCtx(ctx).GasLimit < protocol.MustGetActionCtx(ctx).IntrinsicGas {
 		return nil, action.ErrGasLimit
 	}
+	if !protocol.MustGetFeatureCtx(ctx).EnableWeb3Staking && isWeb3StakingAction(elp) {
+		return nil, errUnsupport
+	}
```

Comment:
the caller needs to assert, if this happens, skip this error and continue (so the tx is silently dropped, and continue to pick other txs), similar to line 460:
```
case action.ErrChainID:
				continue
```